paddle/operators/sequence_pool_op.h

```diff
@@ -135,6 +139,19 @@ class SequencePoolGradKernel : public framework::OpKernel<T> {
           in_g_e.device(place) =
               (out_g_e / std::sqrt(static_cast<T>(h))).broadcast(bcast);
           break;
+        case MAX: {
+          auto in_t = in->Slice<T>(static_cast<int>(lod[i]),
+                                   static_cast<int>(lod[i + 1]));
+          auto out_t = out->Slice<T>(i, i + 1);
```

Comment:
Need to update code, the `Slice` remove the template T: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/framework/tensor.h#L124

Reply:
Done